### DISCUSSION TITLE
Static linking

### DISCUSSION THREAD

**Discussion by MetalInMyVeins** [dankamongmen/notcurses#2777] [Q&A]
Can someone tell me how to statically link notcurses?
I have built notcurses from source and installed in `~/.local`. The dynamically linked binary shows a tons of dependencies. I want to get rid of those.
My distro is Arch. If you have any query, ask me.

**Comment by dankamongmen** [comment]
i thought that the arch package installed the static libraries along with the dynamic, but am not certain (and don't have an arch install handy to check). look for e.g. libnotcurses.a. if you haven't built them, you'll want to build notcurses with `USE_STATIC=yes`.

once you've got them installed, `pkg-config --libs notcurses` ought be sufficient to get the necessary lflags.
